src/schemas/postSchemas.js

```diff
@@ -0,0 +1,4 @@
+import Joi from "joi";
+
+export const sendPostSchema = Joi.string().uri().trim().required();
```

Comment:
The name is misleading, it seems that you are making a schema to the post body but instead you are making one for only 1 specific key. Make a schema for the whole object as i mentioned on the middleware that uses it.